hacktober-web/README.md

```diff
@@ -1,6 +1,18 @@
 # hacktober-web
 
+## First Time Setup
+
+### Requirements
+
+1. [Git](http://git-scm.com/download "Git") - Version Control and connection to Github
+2. [VSCode](https://code.visualstudio.com/download "VSCode") - Code editor (or your preferred editor)
```

Comment:
I'd lead this with "your preferred text editor or IDE", as opposed to following with it. It steers users towards your own preference otherwise.